apps/linode-marketplace-drupal/roles/post/templates/motd.j2

```diff
@@ -0,0 +1,12 @@
+Welcome to your Drupal installation on Linode!
```

Comment:
We should future proof and remove Linode.

Reply:
Reference to "Linode" removed.